build.gradle.kts

```diff
@@ -1,55 +1,43 @@
-buildscript {
-    val kotlinVersion = "1.7.21"
+plugins {
+  `maven-publish`
+}
 
-    repositories {
-        mavenLocal()
-        mavenCentral()
-        jcenter()
-        maven("https://oss.jfrog.org/oss-snapshot-local/")
-    }
+allprojects {
+  version = "1.0.0-SNAPSHOT"
+  group = "com.strumenta.antlr-kotlin"
 
-    dependencies {
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
-    }
-}
+  repositories {
+    mavenCentral()
+  }
 
-// a small hack: the variable must be named like the property
-// jitpack will pass -Pversion=..., so `val version` is required here.
-val version: String by project
-// we create an alias here...
-val versionProperty = version
-// do the same for group
-val group: String by project
-val groupProperty = if (group.endsWith(".antlr-kotlin")) {
-    group
-} else {
-    // just another jitpack hack
-    "$group.antlr-kotlin"
-}
+  apply(plugin = "maven-publish")
 
-allprojects {
-    // ... because `version` is another var here.
-    // when version is hardcoded here, jitpack can not overwrite it.
-    // the default version can now be changed in gradle.properties
-    version = versionProperty
-    group = groupProperty
+  // TODO: setup a real Maven repository to publish to, e.g., Maven Central
```

Comment:
We could publish on Maven Central, we have already an account to publish everything under the com.strumenta group